tests/ui/sprite-frame.test.ts

```diff
@@ -78,11 +97,55 @@ test('spriteframe.clone', () => {
 
     expect(sp_clone.pixelsToUnit).toEqual(sp.pixelsToUnit);
     expect(sp_clone.pivot).toEqual(sp.pivot);
+    sp_clone.pivot.add2f(100,200);
+    expect(sp_clone.pivot !== sp.pivot).toEqual(true);
+
     expect(sp_clone.mesh).toEqual(sp.mesh);
+    sp_clone.mesh?.destroy();
+    expect(sp_clone.mesh !== sp.mesh).toEqual(true);
+    
     expect(sp_clone.uv).toEqual(sp.uv);
+    sp_clone.uv.splice(0, sp_clone.uv.length);
+    expect(sp_clone.uv !== sp.uv).toEqual(true);
+
     expect(sp_clone.unbiasUV).toEqual(sp.unbiasUV);
+    sp_clone.unbiasUV.splice(0, sp_clone.unbiasUV.length);
+    expect(sp_clone.unbiasUV !== sp.unbiasUV).toEqual(true);
+
     expect(sp_clone.uvSliced).toEqual(sp.uvSliced);
+    sp_clone.uvSliced.splice(0, sp_clone.uvSliced.length);
+    expect(sp_clone.uvSliced !== sp.uvSliced).toEqual(true);
+
     expect(sp_clone.vertices).toEqual(sp.vertices);
+    sp_clone.vertices?.rawPosition.splice(0,  sp_clone.vertices?.rawPosition.length);
+    expect(sp_clone.vertices?.rawPosition !== sp.vertices?.rawPosition).toEqual(true)
+    sp_clone.vertices?.positions.splice(0,  sp_clone.vertices?.positions.length);
+    expect(sp_clone.vertices?.positions !== sp.vertices?.positions).toEqual(true);
+    sp_clone.vertices?.indexes.splice(0,  sp_clone.vertices?.indexes.length);
+    expect(sp_clone.vertices?.indexes !== sp.vertices?.indexes).toEqual(true);
+    sp_clone.vertices?.uv.splice(0,  sp_clone.vertices?.uv.length);
+    expect(sp_clone.vertices?.uv !== sp.vertices?.uv).toEqual(true);
+    sp_clone.vertices?.nuv.splice(0,  sp_clone.vertices?.nuv.length);
+    expect(sp_clone.vertices?.nuv !== sp.vertices?.nuv).toEqual(true);
+    sp_clone.vertices?.minPos.add3f(100,200,300);
+    expect(sp_clone.vertices?.minPos !== sp.vertices?.minPos).toEqual(true);
+    sp_clone.vertices?.maxPos.add3f(100,200,300);
+    expect(sp_clone.vertices?.maxPos !== sp.vertices?.maxPos).toEqual(true);;
```

Comment:
I think can treat vertices as a whole. For example, just modify `vertices.rawPosition`, then check vertices again. Don't have to modify every element of vertices.

Reply:
The vertexes are copied separately, and not all copies are necessarily normal.